instructors.md

```diff
@@ -7,196 +7,248 @@ subtitle: Instructor's Guide
+    repos in your Desktop, but under different names, e.g., pretend one is your
+    computer at work: 
+
+    ~~~{.input}    
+    $ git clone https://github.com/vlad/planets.git planets-at-work
+    ~~~
+
+*   It's very common that learners mistype the remote alias or the remote URL
+    when adding a remote, so they cannot `push`. You can diagnose this with
+    `git remote -v` and checking carefully for typos. 
+    - To fix a wrong alias, you can do `git remote rename <old> <new>`. 
+    - To fix a wrong URL, you can do `git remote set-url <alias> <newurl> `.
+
+*   Before cloning the repo, be sure that nobody is inside another repo. The
+    best way to achieve this is moving to the `Desktop` before cloning: `cd &&
+    cd Desktop`.
+
+*   If both repos are in the `Desktop`, have them to clone their collaborator
+    repo under a given directory using a second argument: 
+
+    ~~~{.input}    
+    $ git clone https://github.com/vlad/planets.git vlad-planet`
+    ~~~
+    
+*   The most common mistake is that learners `push` before `pull`ing. If they
+    `pull` afterward, they may get a conflict.
+
+*   Conflicts, sometimes weird, will start to arise. Stay tight: conflicts are
+    next.
 
 ## [Conflicts](07-conflict.html)
 
-FIXME
+*   Expect the learners to make mistakes. Expect *you* to make mistakes. This
+    happens because it is late in the lesson and everyone is tired.
+
+*   If you're the only instructor, the best way to create a conflict is:
+
+    - Clone your repo in a different directory, pretending is your computer at
+      work: `git clone https://github.com/vlad/planets.git planets-at-work`.
+    - At the office, you make a change, commit and push.  
+    - At your laptop repo, you (forget to pull and) make a change, commit and 
+      try to push. 
+    - `git pull` now and show the conflict.
+
+*   Learners usually forget to `git add` the file after fixing the conflict and
+    just (try to) commit. You can diagnose this with `git status`.
+
+*   Remember that you can discard one of the two parents of the merge:
+
+    - discard the remote file, `git checkout --ours conflicted_file.txt`
+    - discard the local file, `git checkout --theirs conflicted_file.txt`
+    
+    You still have to `git add` and `git commit` after this.
 
 ## [Open Science](08-open.html)
 
-FIXME
 
 ## [Licensing](09-licensing.html)
 
-FIXME
+We teach about licensing because:
+
+- questions about who owns what, or can use what, arise naturally once we
+  start talking about using public services like GitHub to store files; and 
+- the discussion gives learners a chance to catch their breath after what
+  is often a frustrating couple of hours.
 
 ## [Hosting](10-hosting.html)
 
-FIXME
+A common concern for learners is having their work publicly available on
+Github.  While we encourage open science, sometimes private repos are the
+only choice. It's always interesting to mention the options to have
```

Comment:
options of having them, and benefits of having them: you can still tie online CI services to private repositories.  Taking advantage of these existing build connections can save a lot of time and effort.